src/science/landuse/landuse3.F90

```diff
@@ -3,7 +3,23 @@ MODULE landuse_variable
   IMPLICIT NONE
 
   SAVE
-
+!*# Overview of landuse3.F90
+!
+! these are three-type landuse-type variable
+!
+! luc%var_x(mland,mvmax): variable indexed by "mland" and "mvmax" before landuse change
+! luc%var_y(mland,mvmax): variable indexed by "mland" and "mvmax" after land use change
+! lucmp%var(mp):          variable indexed by the patch number from 1 to mp
+! \[y=x dot transit\]
```

Comment:
I think you want to replace `dot` with `\cdot` but I'm not sure what operator you actually want. You can see a list here: https://oeis.org/wiki/List_of_LaTeX_mathematical_symbols